iree/vm/BUILD

```diff
@@ -179,40 +178,40 @@ cc_library(
         "bytecode_dispatch_util.h",
         "bytecode_module.c",
         "bytecode_module_impl.h",
-        "bytecode_op_table.h",
     ],
     hdrs = [
         "bytecode_module.h",
+        "generated/bytecode_op_table.h",
     ],
     deps = [
-        ":bytecode_op_table_gen",
         ":vm",
         "//iree/base:api",
         "//iree/base:core_headers",
         "//iree/base:flatcc",
         "//iree/base:tracing",
         "//iree/schemas:bytecode_module_def_c_fbs",
     ],
+    copts = ["-Iiree/vm/generated"],
```

Comment:
What prompted adding this copt? I don't think it should be needed, with the include `#include "iree/vm/generated/bytecode_op_table.h"`

Reply:
I was following https://docs.bazel.build/versions/master/cpp-use-cases.html#adding-include-paths, but I just tested without the copt and it looks like you're right that it is unnecessary